### DISCUSSION TITLE
when i open github on other website, it will open on incoginto tab

### DISCUSSION THREAD

**Discussion by kurisucat** [bromite/bromite#1734] [Q&A]
Version: ` 96.0.4664.54` 
Arch: `arm64`
Android version: `10.0` 
Device model: SAMSUNG SM-G960U

Steps to reproduce the bug:
1. Google "Github"
2. tap to open "githib"
3. see, the github open on incognito

-- or --

1. open heroku
2. link to github
3. see, the github open on incognito

i need bromite open github in normal tab.

and the bot may ban me i take issue. so i ask on there.

**Comment by uazo** [comment]
is not that most likely it is not incognito, but you simply have the **black** top bar?

do you have this symbol at the top?
![image](https://user-images.githubusercontent.com/29201891/150175797-6436e862-ed34-4a3f-b0aa-66ac6a790d89.png)

**Reply by kurisucat** [reply]
thank for your answer.


yes, i make a screen record video.


https://user-images.githubusercontent.com/64016189/150274107-f306856e-0ed3-45be-b795-5b2d8c8806b3.mp4

but i can't record the FLAG_SECURE page.

**Reply by uazo** [reply]
> but i can't record the FLAG_SECURE page.

you can try by changing the flag chrome://flags/#incognito-screenshot

**Reply by kurisucat** [reply]
thx for ur help, i can record incognito tab.

and then it is a new video record.

https://user-images.githubusercontent.com/64016189/150363769-f9743383-48a1-4809-87bf-4ca59710687a.mp4

**Reply by uazo** [reply]
very strange, by any chance you have set the option "Open External Links in incognito"?
if so, would you be able to disable it and try?

**Reply by kurisucat** [reply]
**ok, it is work in normal.**
i'm so sorry i waste your time.
and my english it is poor so i missed it.
as compensation, i'll try to add some translations to prevent confusion for people with poor english

**Reply by uazo** [reply]
however, that behavior is strange.
if you can, try to post the netlog. https://www.chromium.org/for-testers/providing-network-details
